apps/pie-docs/src/assets/styles/components/_tokensTable.scss

```diff
@@ -117,11 +112,8 @@ $tokensTable-item-vertical-padding: var(--dt-spacing-e);
     margin-block-end: 0;
     margin-inline-start: auto;
     margin-inline-end: auto;
-    left: 0;
-    right: 0;
+    inset: calc(var(--base-negative-space) + 8px) 0 calc(var(--base-negative-space) - 8px) 0;
```

Comment:
I think this might be a little more readable splitting it into `inset-inline` and `inset-block`.
What do you think?

Reply:
this was auto-suggested by the stylient version update.